docs/adapters.md

```diff
@@ -1,4 +1,7 @@
-# Hubot Adapters
+---
+permalink: /docs/adapters/index.html
+layout: docs
```

Comment:
You might find [YAML Front Matter Defaults](http://jekyllrb.com/docs/configuration/#front-matter-defaults) helpful in reducing the repetition here.